From: intrigeri <intrigeri@boum.org>
Origin: https://github.com/micahflee/torbrowser-launcher/commit/b4c30f0a29d33cbc1b3140dd4fc10256137fc09e
Date: Mon, 15 Sep 2014 16:48:26 +0000
Subject: AppArmor: include the freedesktop.org abstraction in the
 start-tor-browser profile.

It now needs access to /usr/share/pixmaps/.
---
 apparmor/torbrowser.start-tor-browser | 1 +
 1 file changed, 1 insertion(+)

diff --git a/apparmor/torbrowser.start-tor-browser b/apparmor/torbrowser.start-tor-browser
index f0bc429..1f5fcf1 100644
--- a/apparmor/torbrowser.start-tor-browser
+++ b/apparmor/torbrowser.start-tor-browser
@@ -4,6 +4,7 @@
   #include <abstractions/base>
   #include <abstractions/bash>
   #include <abstractions/fonts>
+  #include <abstractions/freedesktop.org>
 
   capability sys_ptrace,
 
